joomla/joomla-cms
# Add "show global value" feature to com_contact

This PR expands the already merged #11911 to com_contact

### Summary of Changes
Enables the new "Show Global Value" feature for all contact forms.

### Testing Instructions
Test the contact form and the related menu item forms. All list elements with a "Use Global" entry should show the global value like this:
![showglobal](https://cloud.githubusercontent.com/assets/1018684/20215118/a752ef28-a812-11e6-9e90-22da53946e0c.PNG)

Exception: The robots field will not show the global value.

Note: If you get a message that some global values can't be found, try saving the component options.

### Documentation Changes Required
None
